flavors/vpp/vpp_flavor.go

```diff
@@ -71,8 +71,8 @@ func (f *Flavor) Inject() bool {
 
 	f.injectEmbedded()
 
-	f.GoVPP.Deps.PluginInfraDeps = *f.FlavorLocal.InfraDeps("govpp", local.WithConf())
-	f.VPP.Deps.PluginInfraDeps = *f.FlavorLocal.InfraDeps("default-plugins", local.WithConf())
+	f.GoVPP.Deps.PluginInfraDeps = *f.FlavorLocal.InfraDeps("govpp-plugin", local.WithConf())
```

Comment:
I'm not really sure about the changes to plugin names with added `-plugin` suffix. This makes already enormous logs with even longer lines, and same thing with command flags, with all of them set the entire line will be huge. And also I don't think the suffix really serves any purpose since they are all plugins anyway and if we add same suffix to each what added value is there, beside adding extra word?

@milanlenco @lukasmacko @rastislavszabo Thoughts?

Reply:
It's because of command flags. Without it, flags would be called --vpp-config, --linux-config... I thought it can be confusing because it's actually config for the plugin, not the vpp/linux. Adding the suffix to other plugins like govpp is just because of a consistency. The rest of you've written is true.

Reply:
Then I would propose to go with one of two ways:
1. keep this suffix for these two only (vpp/linux), sacrificing little consistency to have shorter names for the rest of plugins/configs/logs
2. or allow config name to be defined explicitly for some plugins

Reply:
I would go for the 1.

Reply:
Agreed. This seems to be best option for time being and also makes it easier transition:
- defaultplugins -> vpp-plugin
- linuxplugin -> linux-plugin
- and other plugins stay same (no need to change flags/configs)